docs/tutorial/using-checkbox/advanced-config.rst

```diff
@@ -0,0 +1,340 @@
+.. _advanced_configs:
+
+.. todo::
+
+   Replace smoke -> tutorial
+
+===============
+Advanced Config
```

Comment:
Having read the material, I agree with @pieqq : this is not a tutorial, but a series of short howtos. Analogous to for example the series of "Manage ..." how-tos under https://juju.is/docs/juju/how-to for juju.